### ISSUE TITLE
How to use suspense mode on specific components?

### ISSUE THREAD

**Issue by nghiepit** [FormidableLabs/urql#690]
Hi all,

I just want to use suspense for specific components.
So, I set `suspense: false` for the urql client and `context.suspense: true`
```js
const client = createClient({
  url: 'http://localhost:1234/graphql',
  suspense: false, // Disable suspense by default
  exchanges: [
    dedupExchange,
    suspenseExchange, // Add suspenseExchange to your urql exchanges
    cacheExchange,
    fetchExchange,
  ],
});
```

I just want to  enable`suspense` mode only on `CommentList.jsx`
```jsx
// CommentList.jsx
const context= {
   suspense: true
}

const ComponentList = () => {
   const [data, fetching] = useQuery({
     query: ...,
     context
   });

  console.log(data, fetching);// undefined true
   ....
}

export default props => (
  <Suspense fallback="loading....">
    <ComponentList {...props} />
  </Suspense>
);
```

Thanks in advance.

**Comment by kitten** [comment]
This isn't well documented (purposefully) but you can disable `suspense` mode selectively by setting it in the context to `false`: https://github.com/FormidableLabs/urql/blob/master/packages/core/src/client.ts#L232

So for instance for `useQuery` you'd have to write:

```js
const context = { suspense: false };

const YourComponent = () => {
  useQuery({
    // ...
    context,
  });

  // ...
};
```

Now the obvious question that follows is, **why is client-side suspense mode hard to enable/disable?**

And to that there are two parts: For us it's always been intended to be used for server-side rendering as a server-side suspense-mode in combination with `react-ssr-prepass`, and the other side is that we're deprecating `@urql/exchange-suspense` and client-side suspense support. We're adding notices for this soon.

The tl;dr reason is that users lose control over the loading behaviour entirely and run into odd issues due to usage fairly frequently. It's also _not a good loading primitive_ on the client-side and hasn't been intended as such by the React team. It's not meant to be used as a "fancy loading mode" 😅 

More details can be found here: https://github.com/FormidableLabs/urql/issues/685#issuecomment-611133478
